### PR TITLE
Dev/yannick.martin/logging level

### PR DESCRIPTION
**Pull request by ymartin-ovh**
Hello !

My primary objective is to disable debug logs.

When I'm running test, despite nameservers details added, I still have issue: permissions seems to break deepequal call but I don't understand why:

Expected map[string]interface {}{"description":"a nameserver description", "id":"0", "name":"mynameserver.example.com", "namesrv1":"1.2.3.4", "permissions":1}

got map[string]interface {}{"description":"a nameserver description", "id":"0", "name":"mynameserver.example.com", "namesrv1":"1.2.3.4", "permissions":1}

If I remove permissions field, check pass.

Regards

### CODE REVIEW

**Comment by pavel-z1** [comment]
Regarding the issue with tests. 
Can you please describe in more details your phpipam subnets/ips or some configuration examples?
It is possible to share your tests? This would make it easier to reproduce the problem.

**Comment by ymartin-ovh** [comment]
I address unit test fix in a dedicated PR: https://github.com/pavel-z1/phpipam-sdk-go/pull/13

**Comment by ymartin-ovh** [comment]
Look and abandon tflog because logger config is in context and we don't have a global context for this.
With apex I achieve same goal as I did with logorus.

Regards

File: phpipam/client/client.go [hunk 1]
```diff
@@ -20,12 +22,20 @@ type Client struct {
 
 // NewClient creates a new client.
 func NewClient(s *session.Session) *Client {
+	log.SetHandler(logfmt.New(os.Stderr))
+
+	SetLevel(log.InfoLevel)
```
Comment (pavel-z1) [inline phpipam/client/client.go:27]: But if you want to implement the logging levels, log level should be passed from the environment variable, if the environment variable does not correspond to one of the desired values, then set default `log.InfoLevel`

File: vendor/vendor.json [hunk 2]
```diff
@@ -2,12 +2,31 @@
...
+			"revisionTime": "2020-12-14T06:45:52Z"
+		},
+		{
+			"checksumSHA1": "zamq7bor1Nt+tXE3sJ3lH21K/f4=",
+			"path": "log",
+			"revision": ""
 		}
 	],
-	"rootPath": "github.com/pavel-z1/phpipam-sdk-go"
+	"rootPath": "phpipam-sdk-go"
```
Comment (pavel-z1) [inline vendor/vendor.json:31]: Return please root path to "github.com/pavel-z1/phpipam-sdk-go"

**Review by pavel-z1 — CHANGES_REQUESTED** [review]
Well, this sdk can be used not only in terraform
So I don't mind using apex log

File: phpipam/client/client.go (hunk 1 above)
Comment (ymartin-ovh) [reply phpipam/client/client.go:27]: Ok I will implement this.

File: vendor/vendor.json (hunk 2 above)
Comment (ymartin-ovh) [reply vendor/vendor.json:31]: Oups, sorry, fixing

File: phpipam/client/client.go (hunk 1 above)
Comment (ymartin-ovh) [reply phpipam/client/client.go]: Tell if me if environment variable name is OK for you.
I don't know where to add some doc (at least environment variable reference).

Regards

Comment (pavel-z1) [reply phpipam/client/client.go]: `PHPIPAMSDK_LOGLEVEL` - is good.
Can you please add information about loglevel to the [README.md](https://github.com/pavel-z1/phpipam-sdk-go/blob/master/README.md) file?

Comment (ymartin-ovh) [reply phpipam/client/client.go]: Description added

**Comment by pavel-z1** [comment]
Thank you
Merged to master

**Pull request merged**